feebleline.el

```diff
@@ -214,6 +216,25 @@ Returns a pair with desired column and string."
   (with-current-buffer feebleline--minibuf
     (erase-buffer)))
 
+
+(defun feebleline--count-windows ()
+  "Total window count."
+  (let ((count 0))
+    (dolist (fn (frame-list))
+      (setq count (+ (length (window-list fn)) count)))
+    count))
```

Comment:
Just to make sure, this function count all windows across all frames. :D That's why I don't use normal count windows, and I don't know if there is a default way of doing this... :(